app/views/home/index.html.erb

```diff
@@ -1,2 +1,33 @@
-<h1>Home#index</h1>
-<p>Find me in app/views/home/index.html.erb</p>
+<body class="login cover-container">
```

Comment:
@rashadmad There's currently a body tag in application.html.erb. I don't know if this body tag might cause any conflict with that, but you might want to double check that.